### PR TITLE
chore: migrate to owl bot

### PR DESCRIPTION
**Pull request by parthea**

### CODE REVIEW

File: CONTRIBUTING.rst [hunk 1]
```diff
@@ -21,8 +21,8 @@ In order to add a feature:
 - The feature must be documented in both the API and narrative
   documentation.
 
-- The feature must work fully on the following CPython versions:  2.7,
-  3.5, 3.6, 3.7 and 3.8 on both UNIX and Windows.
+- The feature must work fully on the following CPython versions:
```
Comment (busunkim96) [inline CONTRIBUTING.rst:24]: I think this library still supports 2.7 so the CONTRIBUTING.rst may need to be excluded temporarily.

File: setup.py [hunk 2]
```diff
@@ -17,6 +17,20 @@
 
 import setuptools
 
+# Disable version normalization performed by setuptools.setup()
```
Comment (busunkim96) [inline setup.py:20]: Is this intentionally being rolled into this PR?

Comment (parthea) [reply setup.py:20]: This line is already in master. I think the branch is out of date.

Comment (parthea) [reply setup.py:20]: I'm going to retry the migration.

**Review by tseaver — COMMENTED** [review]
LGTM with the exception of `CONTRIBUTING.rst`, as pointed out by @busunkim96

File: setup.py (hunk 2 above)
Comment (parthea) [reply setup.py]: This issue was due to https://github.com/googleapis/repo-automation-bots/issues/1752. I'm going to force push the branch.

**Comment by parthea** [comment]
I'm going to force push this branch again due to issue googleapis/repo-automation-bots#1752. I'll also address the review comments at the same time.

File: noxfile.py [hunk 3]
```diff
@@ -129,14 +154,28 @@ def system(session):
         "google-cloud-iam",
         "google-cloud-pubsub < 2.0.0",
         "google-cloud-kms < 2.0dev",
+        "-c",
+        constraints_path,
```
Comment (tseaver) [inline noxfile.py:158]: Ugh, this change is now in conflict with #433, which works around the breakage incurred by `googleapis-common-protos` dropping Python 2.7 support.  Likely we need to tweak `synth.py` to preserve that change.

**Review by tseaver — CHANGES_REQUESTED** [review]

File: noxfile.py (hunk 3 above)
Comment (parthea) [reply noxfile.py]: Done.

File: CONTRIBUTING.rst (hunk 1 above)
Comment (parthea) [reply CONTRIBUTING.rst]: Done.

**Comment by parthea** [comment]
@tseaver  PTAL

**Review by tseaver — APPROVED** [review]

**Pull request merged**